src/realtimecontroller/RealtimeDataMessage.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+type RealtimeDataMessage = Uint8Array | string;
```

Comment:
The original was `Unit8Array | string | any` right?

Reply:
Correct. I just left types types that are compatible with `WebSocket.send()` [signature](https://developer.mozilla.org/en-US/docs/Web/API/WebSocket/send).

Reply:
We support `any` for JSON (https://github.com/aws/amazon-chime-sdk-js/blob/master/src/task/SendAndReceiveDataMessagesTask.ts#L68) so I don't think we can change that.
It is also a breaking change for the library.

Reply:
Generally, I planned to get rid of `any` type to have strict types.
And set up messages type in one place.
Now I see it can be enhanced with a generic type of message like so:
```ts
type MyMessageType = { subject: string, message: string };
const data = { subject: 'test', message: 'test message' };
context = new AudioVideoControllerState<MyMessageType>(); // can accept any as well
context.realtimeController.realtimeSendDataMessage(topic, data, 100);
```
Would you like me to do that or just bring `any` back?